__tests__/http/controllers/AccountTypesController.test.js

```diff
@@ -0,0 +1,171 @@
+const AccountTypes = require('@http/controllers/AccountTypesController')
+const AccountTypeService = require('@services/AccountTypesService')
+const httpMocks = require('node-mocks-http')
+const { StatusCodes } = require('http-status-codes')
+const Boom = require('@hapi/boom')
+
+AccountTypeService.create = jest.fn()
+AccountTypeService.update = jest.fn()
+AccountTypeService.fetchAll = jest.fn()
+AccountTypeService.fetchById = jest.fn()
+AccountTypeService.deleteById = jest.fn()
+
+const errorMessage = 'Error message'
+const BoomNotFoundError = Boom.notFound(errorMessage)
+const BoomBadDataError = Boom.badData(errorMessage)
+
+let req, res, next
```

Comment:
It is better to wrap things inside a `describe` block so you do not pollute the global namespace. So, I would wrap everything inside a describe like these vars and the beforeEach

Reply:
ok